packages/dusk/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+from spack import *
+import sys
+
+class Dusk(PythonPackage):
+    """A minimal and lightweight front-end for dawn."""
+
+    homepage = 'https://github.com/dawn-ico/dusk'
+    git      = 'https://github.com/dawn-ico/dusk'
+    maintainers = ['BenWeber42']
+
+    version('master', branch='master')
+    
+    extends('python@3.8.0:3.8.999')
+
+    depends_on('dawn4py', type='run')
```

Comment:
```suggestion
    depends_on('dawn4py', type=('build', 'run'))
```
You're probably right that we need to specify it as both build and run dependency (default is build and link, but link doesn't really make much sense for python).

Reply:
This confuses me. Is there such a thing as _building_ in python? :)

Reply:
Well not really, but `setuptools` has a bunch of commands including `build` and `install`. And spack will run those commands (https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#phases).

For dusk `python setuptools build` will basically just collect & copy all dusk python files into a `build` folder (other packages will probably do more during this phase, probably dawn4py). So in a sense dusk has a _building_ phase :sweat_smile: